numba4jax/_src/backends/_cuda.py

```diff
@@ -1,17 +1,22 @@
+import sys
+
 from numba import cuda as ncuda
 from cffi import FFI
 
 from numba import types as nb_types
 
-from numba4jax._src.c_api import find_path_of_symbol_in_library
-
 try:
     _libcuda = ncuda.driver.find_driver()
 
-    libcuda_path = find_path_of_symbol_in_library(_libcuda.cuMemcpy)
+    if sys.platform == "win32":
+        import ctypes.util
+        libcuda_path = ctypes.util.find_library(_libcuda._name)
+    else:
+        from numba4jax._src.c_api import find_path_of_symbol_in_library
+        libcuda_path = find_path_of_symbol_in_library(_libcuda.cuMemcpy)
 
     numba_cffi_loaded = True
-except:
+except ValueError:
```

Comment:
Then it will throw an error that we don't know how to handle at this point. I think a bare `except` is a bad practice because is will 'eat' even keyboard interrupt.